plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -127,15 +129,35 @@ public PluginManager(Config cfg) {
         skipFailedPlugins = cfg.isSkipFailedPlugins();
         hashFunction = cfg.getHashFunction();
         httpClient = null;
+        userAgentInformation = this.getUserAgentInformation();
     }
 
+    private String getUserAgentInformation() {
+        String userAgentInformation="";
```

Comment:
```suggestion
        String userAgentInformation= "Jenkins plugin manager/Dev";
```

Reply:
> can you format your changes please, any IDE should format it relatively well, at least spaces before brackets

I am using vscode, I will keep this in mind.